applications/NEFTME.md

```diff
@@ -0,0 +1,355 @@
+
+https://github.com/LuisNeves05
+https://github.com/andreasvilela
+https://github.com/gmarquesetr
+
+### Team LinkedIn Profiles (if available)
+
+
+https://www.linkedin.com/in/joaoneftme/
+https://www.linkedin.com/in/valtersilva93/
+https://www.linkedin.com/in/andreasvilela/
+https://www.linkedin.com/in/jo%C3%A3o-rodrigues-a22150153/
+https://www.linkedin.com/in/gmarquesetr/
+https://www.linkedin.com/in/luisneves05/
+https://www.linkedin.com/in/marcopsbarbosa/
+https://www.linkedin.com/in/afonsobarbosa/
+
+
+## Development Status :open_book:
+
+ 
+https://github.com/NEFTME
+https://github.com/NEFTME/neftme-poc-dapp
+https://github.com/NEFTME/neftme-poc-api
+
+
+## Development Roadmap :nut_and_bolt:
+
+
+We are currently on Alpha 4 testing phase od our dAPP, and raising a Pre-seed round so we can get to Beta version in the end of the year.
+
+Here goes our Roadmap, and our token pre-seed raise map, with all the milestones in it.
+
+Roadmap
+https://docs.google.com/document/d/1mpJ4rI7My_YO5JP5JGW3M7QRq-cMIZzLw6sjVVk8_dU/edit?usp=sharing
+ 
+https://docs.google.com/spreadsheets/d/1pPKar2YDZnf7FK5dluL2FmPGsG16an0hcKRNvfxOsLE/edit?usp=sharing
+Milestone 1 - Alpha 5
+·       
+Direct Messaging;
+·       
+NFT as a Audio;
+·       
+Abuse Content analysis and call to action;
+·       
+Perks;
+·       
+Share NFTs (link);
+·       
+Share Profile (link).
+Milestone 2 - Beta
+NFT as a Video;
+Invite Code system to control amount of new users;
+Reward system / Referral system by user action;
+Internal wallet;
+On Ramp;
+Challenge “Be real”.
+
+### Overview
+
+- **Total Estimated Duration:** Duration of the whole project (e.g. 2 months)
+- **Full-Time Equivalent (FTE):**  Average number of full-time employees working on the project throughout its duration (see [Wikipedia](https://en.wikipedia.org/wiki/Full-time_equivalent), e.g. 2 FTE)
+- **Total Costs:** Requested amount in USD for the whole project (e.g. 12,000 USD). Note that the acceptance criteria and additional benefits vary depending on the [level](../README.md#level_slider-levels) of funding requested. This and the costs for each milestone need to be provided in USD; if the grant is paid out in Bitcoin, the amount will be calculated according to the exchange rate at the time of payment.
+
+### Milestone 1 — Alpha 5
+
+- **Estimated duration:** 3 months
+- **Costs:** 35K USD
+ 
+
+ | Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| 1. | Direct Messaging | Allow direct decentralized messaging inside the dApp  |
+| 2. | NFT as an Audio | Users are now able to create audio NFTs with their phones |
+| 3. | Abuse content analysis | Ability to report NFTs |
+| 4. | Share NFTs | Share NFTs via a link that redirects to the app |
+| 5. | Share Profiles | Share Profiles via link that redirects to the app |
+| 6. | Smart contracts fixes | Fix smart contract bugs and update staking model |
```

Comment:
@LuisNeves05 you're still missing the mandatory deliverables, so feel free to add these as well to M1 and M2. Keep in mind that deliverables 0a-0d are [mandatory](https://github.com/w3f/Grants-Program/blob/master/applications/application-template.md#development-roadmap-nut_and_bolt) for all milestones, and deliverable 0e at least for the last one.